### ISSUE TITLE
pt_data_inventory

### ISSUE THREAD

**Issue by curtisKJ** [metrumresearchgroup/pmtables#220]
# Summary

I'm a little confused by the summaries for BLQ summaries using `pt_data_inventory`.
The table reports OBS (number of non-BQLs) and BQL (number of BQLs), however, the percentages reported are not what I'd expect. I would expect the calculation of % for OBS to be OBS/(Total samples), not OBS/(Total of OBS); refer to Slack exchange for additional details.

**Comment by curtisKJ** [comment]
@KatherineKayMRG @callistosp

**Comment by kylebaron** [comment]
Yeah; I think that's right.  We pass the blq info when calculating `n_missing` but not to `n_non_missing`.  Just need to get more precise with this along same lines as #209
